tox.ini

```diff
@@ -9,7 +9,13 @@ envlist=
 [flake8]
 max-line-length= 100
 exclude=
-ignore=E252
+ignore =
+    # E252 missing whitespace around parameter equals
+    # Already used too much in the codebase at the point of introduction.
+    E252,
```

Comment:
Maybe open a "Good First Issue" to clean all these up? I agree it probably belongs in its own PR, if it touches a lot of lines.

Reply:
Hundreds!